tests/error/iter_errors/end_missing.py

```diff
@@ -1,12 +1,12 @@
 from guppylang.decorator import guppy
-from guppylang.hugr import tys
 from guppylang.module import GuppyModule
+from guppylang.tys.ty import NoneType
 
 
 module = GuppyModule("test")
 
 
-@guppy.type(module, tys.TupleType(inner=[]))
+@guppy.type(module, NoneType().to_hugr())
```

Comment:
hmmm, perhaps we should have (somewhere outside test)
```
HUGR_UNIT_TYPE = TupleType([]).to_hugr()
```
and then `NoneType.to_hugr()` just returns that - might possibly be a bit clearer

Reply:
Hmm not sure if this is clearer. In `TupleType` we're now doing a similar thing by calling `to_hugr()` on a `SumType` constructed in-place. But I don't really feel strongly about this.

Btw since types are immutable, we could drive this further and just intern all types. But this sounds like premature optimisation

Reply:
Ok, fair enough, this is clear enough. Certainly no need to intern everything! :)

Reply:
Ok, fair enough, this is clear enough. Certainly no need to intern everything! :)